pkg/controller/gitopsconfig/controller.go

```diff
@@ -19,6 +19,7 @@ package gitopsconfig
 import (
 	"context"
 	goerrors "errors"
+	"golang.org/x/xerrors"
```

Comment:
package xerrors is not in Go standard library, so it should go in the block between l.25-l.42, here and in all files below